src/redux/editOptions.js

```diff
@@ -1,262 +1,28 @@
-  }
-
-  analytics.identify(null, {
-    hiddenCoins: updatedHiddenCoins,
-    pinnedCoins: updatedPinnedCoins,
-  });
-  savePinnedCoins(updatedPinnedCoins, accountAddress, network);
-  dispatch({
-    payload: {
-      currentAction: EditOptions.none,
-      hiddenCoins: updatedHiddenCoins,
-      pinnedCoins: updatedPinnedCoins,
-      recentlyPinnedCount: recentlyPinnedCount + 1,
-      selectedCoins: [],
-    },
-    type: SET_PINNED_COINS,
-  });
-};
-
-export const addCoinsToHiddenList = additionalCoinsToHide => (
-  dispatch,
-  getState
-) => {
-  const { accountAddress, network } = getState().settings;
-  const {
-    hiddenCoins,
-    pinnedCoins,
-    recentlyPinnedCount,
-  } = getState().editOptions;
-  const updatedPinnedCoins = pinnedCoins;
-  const updatedHiddenCoins = uniq(
-    concat(hiddenCoins, without(additionalCoinsToHide, ...pinnedCoins))
-  );
-  // Nothing to update
-  if (hiddenCoins.length === updatedHiddenCoins.length) {
-    return;
-  }
-
-  analytics.identify(null, {
-    hiddenCoins: updatedHiddenCoins,
-    pinnedCoins: updatedPinnedCoins,
-  });
-  saveHiddenCoins(updatedHiddenCoins, accountAddress, network);
-  dispatch({
-    payload: {
-      currentAction: EditOptions.none,
-      hiddenCoins: updatedHiddenCoins,
-      pinnedCoins: updatedPinnedCoins,
-      recentlyPinnedCount: recentlyPinnedCount + 1,
-      selectedCoins: [],
-    },
-    type: SET_HIDDEN_COINS,
-  });
-};
-
-export const setHiddenCoins = () => (dispatch, getState) => {
-  const { accountAddress, network } = getState().settings;
-  const {
-    currentAction,
-    hiddenCoins,
-    pinnedCoins,
-    recentlyPinnedCount,
-    selectedCoins,
-  } = getState().editOptions;
-  let updatedPinnedCoins = pinnedCoins;
-  let updatedHiddenCoins = hiddenCoins;
-  if (
-    currentAction === EditOptions.standard ||
-    currentAction === EditOptions.unpin
-  ) {
-    updatedPinnedCoins = without(pinnedCoins, ...selectedCoins);
-    savePinnedCoins(updatedPinnedCoins, accountAddress, network);
-    updatedHiddenCoins = union(selectedCoins, hiddenCoins);
-  } else if (currentAction === EditOptions.unhide) {
-    updatedHiddenCoins = without(hiddenCoins, ...selectedCoins);
-  }
-
-  analytics.identify(null, {
-    hiddenCoins: updatedHiddenCoins,
-    pinnedCoins: updatedPinnedCoins,
-  });
-  saveHiddenCoins(updatedHiddenCoins, accountAddress, network);
-  dispatch({
-    payload: {
-      currentAction: EditOptions.none,
-      hiddenCoins: updatedHiddenCoins,
-      pinnedCoins: updatedPinnedCoins,
-      recentlyPinnedCount: recentlyPinnedCount + 1,
-      selectedCoins: [],
-    },
+    payload: coins,
     type: SET_HIDDEN_COINS,
   });
 };
 
 // -- Reducer ----------------------------------------- //
 const INITIAL_STATE = {
-  currentAction: EditOptions.none,
   hiddenCoins: [],
-  isCoinListEdited: false,
-  isCoinListEditedValue: new Value(0),
-  pinnedCoins: ['eth'],
-  recentlyPinnedCount: 0,
```

Comment:
this value is currently still used in `BalanceCoinRow`

Reply:
we still use this in one selector.

Reply:
removed, thanks!